sdk/src/models/record-scanner/recordsFilter.ts

```diff
@@ -0,0 +1,20 @@
+import { RecordSearchParams } from "../record-provider/recordSearchParams";
+
+/**
+ * RecordsFilter is an extension of RecordSearchParams that represents a filter for scanning encrypted or owned records.
+ * 
+ * @example
+ * const recordsFilter: RecordsFilter = {
+ *     start: 0,
+ *     end: 100,
+ *     program: "...",
+ *     record: "...",
```

Comment:
```suggestion
 *     program: "credits.aleo",
 *     record: "credits",
```

Same here, we should use actual values.

Reply:
done